generation/README.md

```diff
@@ -287,10 +287,17 @@ plugin:
   enable_dev: true
 ```
 
-3. Start the local perses instance, from the perses root directory:
+3. Start your local Perses instance. You can do this in one of two ways:
+
+   - **Option 1: From source**
+     1. Follow the official instructions to build and run Perses from source: [Install from Source](https://perses.dev/perses/docs/installation/from-source/)
+
+   - **Option 2: Using Docker**
+     1. Follow the official instructions to run Perses in a Docker container: [Install in a Container](https://perses.dev/perses/docs/installation/in-a-container/)
+
 ```bash
-./scripts/api_backend_dev.sh  
-```
+# From the Perses root directory, start the backend server:
+./scripts/api_backend_dev.sh
```

Comment:
since you linked the way to install, you don't need anymore to say how to start Perses. Specially because this script is part of the repo `perses/perses` and doesn't make sense when you are installing Perses using docker for example.

Reply:
it doesn't make sense also because this script is starting the binary with the perses dev config containing in the `perses/perses` repo.

Reply:
> since you linked the way to install, you don't need anymore to say how to start Perses. Specially because this script is part of the repo `perses/perses` and doesn't make sense when you are installing Perses using docker for example.

Thank you for the pointer. done in 0a7495856ff2c27f451f50fe976ded6174beb95f

Reply:
> you also need to remove or replace the line 23 in this file: `- You have downloaded [Perses](https://github.com/preses/perses)`

good find. Done in e56513d9c3869f71629f35ea78acc7ed5e47fa45